### PR TITLE
Enable automatic release

### PR DESCRIPTION
**Pull request by negokaz**
## Employed sbt plugin

[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release)

## Changes other than code

- **Set secrets**
  - https://github.com/lerna-stack/akka-entity-replication/settings/secrets/actions
- **Published PGP public key**
  - http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x8fdbd95f28589bcc746b7c7d7057ae017f2eacee
- **Added a tag to master to attach a version larger than `v0.1.0-SNAPSHOT`**
   - [v0.1.1](https://github.com/lerna-stack/akka-entity-replication/tree/v0.1.1)

## Testing

branch: [auto-release-test](https://github.com/lerna-stack/akka-entity-replication/tree/auto-release-test)

- **GitHub Action: release job**
  - https://github.com/lerna-stack/akka-entity-replication/runs/1699234436?check_suite_focus=true
- **Badge on README**
  - image

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: Release
```
Comment (negokaz) [inline .github/workflows/release.yml:1]: [memo] Please refer:

> GitHub Actions
>
> **[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release#github-actions)**

File: build.sbt [hunk 2]
```diff
@@ -12,7 +12,6 @@ lazy val lerna = (project in file("."))
     inThisBuild(
       List(
         scalaVersion := "2.12.12",
-        version := "0.1.0-SNAPSHOT",
```
Comment (negokaz) [inline build.sbt:15]: [memo]

> version: handled by sbt-dynver
>
> **[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release#:~:text=version:,dynver)**

File: publish.sbt [hunk 3]
```diff
@@ -2,12 +2,6 @@ ThisBuild / organization := "io.github.lerna-stack"
 ThisBuild / organizationName := "Lerna Project"
 ThisBuild / organizationHomepage := Some(url("https://lerna-stack.github.io/"))
 
-ThisBuild / scmInfo := Some(
```
Comment (negokaz) [inline publish.sbt:5]: [memo] 

> sbt-git: to automatically populate scmInfo
>
> **[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release#:~:text=sbt%2dgit,populate%20scmInfo)**

File: publish.sbt [hunk 4]
```diff
@@ -23,9 +17,3 @@ ThisBuild / homepage := Some(url("https://github.com/lerna-stack/akka-entity-rep
 
 // Remove all additional repository other than Maven Central from POM
 ThisBuild / pomIncludeRepository := { _ => false }
-ThisBuild / publishTo := {
```
Comment (negokaz) [inline publish.sbt:26]: [memo]

> publishTo: handled by sbt-ci-release
>
> **[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release#:~:text=publishTo:,release)**

File: publish.sbt [hunk 5]
```diff
@@ -23,9 +17,3 @@ ThisBuild / homepage := Some(url("https://github.com/lerna-stack/akka-entity-rep
...
-  val nexus = "https://oss.sonatype.org/"
-  if (isSnapshot.value) Some("snapshots" at nexus + "content/repositories/snapshots")
-  else Some("releases" at nexus + "service/local/staging/deploy/maven2")
-}
-ThisBuild / publishMavenStyle := true
```
Comment (negokaz) [inline publish.sbt:31]: [memo]

> publishMavenStyle: handled by sbt-ci-release
>
> **[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release#:~:text=publishMavenStyle:,release)**

File: project/plugins.sbt [hunk 6]
```diff
@@ -1,5 +1,4 @@
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "2.0")
-addSbtPlugin("com.jsuereth"   % "sbt-pgp"      % "2.0.1")
```
Comment (negokaz) [inline project/plugins.sbt:2]: [memo]

> By installing sbt-ci-release the following sbt plugins are also brought in:
> 
> sbt-dynver: sets the version number based on your git history
> sbt-pgp: to cryptographically sign the artifacts before publishing
> sbt-sonatype: to publish artifacts to Sonatype
> sbt-git: to automatically populate scmInfo
>
> **[olafurpg/sbt-ci-release: sbt plugin to automate Sonatype releases from Travis CI](https://github.com/olafurpg/sbt-ci-release#:~:text=By%20installing,populate%20scmInfo)**

Comment (xirc) [reply project/plugins.sbt:2]: LGTM

File: build.sbt (hunk 2 above)
Comment (xirc) [reply build.sbt:15]: LGTM

File: publish.sbt (hunk 4 above)
Comment (xirc) [reply publish.sbt:26]: LGTM

File: publish.sbt (hunk 5 above)
Comment (xirc) [reply publish.sbt:31]: LGTM

File: publish.sbt (hunk 3 above)
Comment (xirc) [reply publish.sbt:5]: LGTM

File: .github/workflows/release.yml (hunk 1 above)
Comment (xirc) [reply .github/workflows/release.yml:1]: LGTM

> Note that the tag version MUST start with v
> https://github.com/olafurpg/sbt-ci-release#git

**Review by xirc — APPROVED** [review]
It's awesome 👍

**Pull request merged**